packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -29,39 +31,93 @@ export type AsyncTask = Task & {
 };
 
 export class InteractionTracker {
+  private readonly tracingZones: { [index: string]: Zone } = {};
   constructor() {
+    const interactionTracker: InteractionTracker = this;
+
     const runTask = Zone.prototype.runTask;
     Zone.prototype.runTask = function(
       task: AsyncTask,
       applyThis: unknown,
       applyArgs: unknown
     ) {
-      if (task.eventName && task.eventName.toString().indexOf('click') !== -1) {
-        console.log('Running task');
+      const time = Date.now();
+
+      console.warn('Running task');
+      console.log(task.zone);
+
+      let taskZone = this;
+      if (isTrackedElement(task)) {
         console.log('Click detected');
+
+        const zoneName = randomSpanId();
```

Comment:
What would you think about using the `traceId` as the zoneName rather than the spanId? Generally spans are within a given trace, so the traceId would be the top level ID. Also, since it would just be the trace Id, could we call the zone key simply `traceId`?